src/element/textWysiwyg.tsx

```diff
@@ -102,7 +102,10 @@ export function textWysiwyg({
 
   if (onChange) {
     editable.oninput = () => {
-      onChange(trimText(editable.innerText));
+      onChange({
+        text: trimText(editable.innerText),
+        font: editable.style.font,
+      });
```

Comment:
not clear why we're send `font` from within the wysiwyg. That should go from top down.

Reply:
https://github.com/excalidraw/excalidraw/pull/1356/files#r406773625